api-gateway.yml

```diff
@@ -0,0 +1,79 @@
+openapi: "3.0.1"
+info:
+  title: Github for Jira Gateway
+  description: It routes things
+  version: "0.0.1"
+
+paths:
+  # GITHUB WEBHOOKS
+  /webhooks/{event+}:
+    parameters:
+      - name: "event+"
+        in: "path"
+        description: "Generated path parameter for event+"
+        required: true
+        schema:
+          type: "string"
+    post:
+      responses:
+        204:
+          description: "Default response for POST webhooks"
+      x-amazon-apigateway-integration:
+        credentials: "{{RoleARN}}"
+        httpMethod: "POST"
+        uri: "arn:aws:apigateway:{{WebhooksQueueRegion}}:sqs:path/{{AccountNumber}}/{{WebhooksQueueName}}"
```

Comment:
SQS has a limit of 256 Kb . Are we sure our requests are less than this?

Reply:
As of yet, they have been. I think the largest I've seen was around 10kb.  256kb in text is pretty massive for a webhook.

Reply:
Do we have some metrics around this? We should double-check this imho, otherwise the change looks risky